src/solc_0.8/common/fx-portal/FxBaseRootTunnelUpgradeable.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.2;
+
+import "@maticnetwork/fx-portal/contracts/tunnel/FxBaseRootTunnel.sol";
+
+abstract contract FxBaseRootTunnelUpgradeable is FxBaseRootTunnel {
+    // solhint-disable-next-line no-empty-blocks
+    constructor() FxBaseRootTunnel(address(0), address(0)) {}
```

Comment:
should we add an initializer / disableInitializer on the constructor ?

Reply:
will require Initializable import, I am not sure we want to include this. Will review

Reply:
As this contract is inherited, I believe this is not necessary - @mvanmeerbeck

Reply:
We can add the constructor line in the tunnel